cuenca/resources/card_transactions.py

```diff
@@ -0,0 +1,35 @@
+import datetime as dt
+from dataclasses import dataclass
+from typing import ClassVar, List, cast
+
+from cuenca_validations.types import (
+    CardErrorType,
+    CardNetwork,
+    CardTransactionType,
+    CardType,
+)
+
+from .base import Transaction
+from .resources import retrieve_uri
+
+
+@dataclass
+class CardTransaction(Transaction):
+    _resource: ClassVar = 'card_transactions'
+
+    type: CardTransactionType
+    network: CardNetwork
+    related_card_transaction_uris: List[str]
+    card_last4: str
+    card_type: CardType
+    merchant: str
```

Comment:
Change this to `descriptor`. `merchant` can be determined client side (as a property)

Reply:
`descriptor` is also more consistent with other transactions